easybuild/easyconfigs/m/MPICH/MPICH-3.0.4-GCC-4.8.1-slurm.eb

```diff
@@ -16,7 +16,7 @@ source_urls = ['http://www.mpich.org/static/tarballs/%(version)s']
 preconfigopts = 'export FC="$F90"; export FCFLAGS="$F90FLAGS"; unset F90; unset F90FLAGS; '
 
 # Build shared libraries
-configopts = '--enable-shared'
+configopts = '--enable-shared --with-slurm=/usr/lib64/slurm --with-pmi=pmi2 '
```

Comment:
you're changing an existing easyconfig here, please don't, i.e. make sure MPICH-3.0.4-GCC-4.8.1.eb is not being removed